base/system/format/lang/zh-CN.rc

```diff
@@ -26,7 +26,7 @@ BEGIN
     STRING_FMIFS_FAIL "不能定位 FMIFS 入口点。\n\n"
     STRING_UNKNOW_ARG "未知参数：%s\n"
     STRING_DRIVE_PARM "必要的驱动器参数缺失。\n\n"
-    STRING_ERROR_DRIVE_TYPE "无法获得驱动器类型"
+    STRING_ERROR_DRIVE_TYPE "无法获得驱动器类型.\n"
```

Comment:
For both Chinese files: the full stop must be the round circle character (see the line just above)

Reply:
Done.